### DISCUSSION TITLE
Time for a 1.0 release?

### DISCUSSION THREAD

**Discussion by diazona** [pytest-dev/pytest-localserver#65] [General]
While I was thinking about #46, it occurred to me that regardless of whether what happens to pytest-localserver in the future, it would probably be a good idea to have stable (1.0) release on the books. After all, if we continue developing this package, then surely it would be a reasonable goal to get it to 1.0 eventually, and if we don't continue developing it, I'm sure some people will continue to use it regardless and it would probably give them more confidence in using it if there is a 1.0 release available. And honestly, the package _is_ pretty stable in its current state; there are no known bugs, very few important features to be added, and no reason I can think of to avoid moving on to version 1.0. (I think it would be handy to implement #43 and #47 before version 1 but those won't be hard.)

So @pytest-dev/pytest-localserver-developers, what do you think, should we make the next release of this package version 1.0 instead of 0.8? Or we could just release 0.8 right away and then have the next one following that be 1.0.

**Comment by diazona** [comment]
If there's no input/objections from anyone else, I'm going to go ahead and release 0.8 later this week, and then look toward making the next release 1.0.

**Comment by nicoddemus** [comment]
Sounds good, thanks @diazona. :+1:

**Comment by diazona** [comment]
I guess I had been thinking the 1.0 release would have come out by now (it's been a year since 0.8), but there are still a few features I'd like to get in before marking the package officially stable. They shouldn't be complicated, but realistically I don't see those being completed in the immediate future. Since there have been some contributions in the past year I think it makes sense to make a 0.9 release with the current state of the package. Any thoughts @pytest-dev/pytest-localserver-developers? Unless there are any objections, I can take care of it when I get a chance (maybe this week, maybe next week).

**Reply by RonnyPfannschmidt** [reply]
i recommend pushing a release - simply for the feedback

**Comment by diazona** [comment]
I'm going to wrap this up by saying I am still looking toward doing a 1.0 release, but there are a bunch of changes I'd like to get in before I do that, so I'll continue making regular minor version releases (no less frequently than once a year-ish) until the changes for 1.0 are ready.
